components-core/src/main/java/com/adyen/checkout/components/core/internal/ui/model/ButtonComponentParamsMapper.kt

```diff
@@ -26,7 +26,11 @@ class ButtonComponentParamsMapper(
         )
         return ButtonComponentParams(
             commonComponentParams = commonComponentParamsMapperData.commonComponentParams,
-            isSubmitButtonVisible = (componentConfiguration as? ButtonConfiguration)?.isSubmitButtonVisible ?: true,
+            isSubmitButtonVisible = if (commonComponentParamsMapperData.commonComponentParams.isCreatedByDropIn) {
```

Comment:
Minor: you can declare `val commonComponentParams = commonComponentParamsMapperData.commonComponentParams` before to make this and line above shorter